package.json

```diff
@@ -57,7 +57,8 @@
     "shortid": "2.2.16",
     "sqlite3": "5.0.2",
     "swagger-ui-express": "4.1.6",
-    "typeorm": "0.2.32"
+    "typeorm": "0.2.32",
+    "webdav": "4.3.0"
```

Comment:
We should be able to do the PUT and DELETE requests ourselves instead of using the external library.

Reply:
done